rotkehlchen/accounting/cost_basis.py

```diff
@@ -150,6 +170,7 @@ def __init__(
             msg_aggregator: MessagesAggregator,
     ) -> None:
         self._taxfree_after_period: Optional[int] = None
+        self._accounting_method: str = 'FIFO'
```

Comment:
It would be cool to turn this into an Enum instead of a simple `str`. See other enums in the code to see how it can be done.

Reply:
I gave it a try...